site/src/pages/index.jsx

```diff
@@ -34,6 +34,7 @@ export default function Home() {
                 <ThemedImage
                   alt="Create EC2 Instance"
                   className='shadow--tl browser-content'
+                  onClick={() => window.location.href="https://app.iasql.com#/button/INSERT%20INTO%20instance%20%28ami%2C%20instance_type%2C%20tags%29%0A%20%20VALUES%20%28%27resolve%3Assm%3A%2Faws%2Fservice%2Fcanonical%2Fubuntu%2Fserver%2F20.04%2Fstable%2Fcurrent%2Famd64%2Fhvm%2Febs-gp2%2Fami-id%27%2C%20%27t2.micro%27%2C%20%27%7B%22name%22%3A%22i-1%22%7D%27%29%3B%0A%0AINSERT%20INTO%20instance_security_groups%20%28instance_id%2C%20security_group_id%29%20SELECT%0A%20%20%28SELECT%20id%20FROM%20instance%20WHERE%20tags%20-%3E%3E%20%27name%27%20%3D%20%27i-1%27%29%2C%0A%20%20%28SELECT%20id%20FROM%20security_group%20WHERE%20group_name%3D%27default%27%20AND%20vpc_id%20%3D%20%28SELECT%20id%20FROM%20vpc%20WHERE%20is_default%20%3D%20true%29%29%3B"}
```

Comment:
The query (in the URL and the image) needs to be updated very slightly for the multi-region world we're now in. The final `WHERE` clause also needs `AND region = default_aws_region()`